henrietta_chow/week01/mta/js/main.js

```diff
@@ -0,0 +1,101 @@
+var mta = {
+  N: ['Times Square', '34th', '28th', '23rd', 'Union Square', '8th'],
+  L: ['8th', '6th', 'Union Square', '3rd', '1st'],
+  6: ['Grand Central', '33rd', '28th', '23rd', 'Union Square', 'Astor Place'],
+
+  getStationIndex: function(line, station) {
+    return this[line].indexOf( station ); //instead of running a loop to return index
+  },
+
+  planTrip: function(startLine, startStation, endLine, endStation) {
+    var startIndex = this.getStationIndex(startLine, startStation);
+    var endIndex = this.getStationIndex(endLine, endStation);
+    var startLineUnionSqIndex = this.getStationIndex(startLine, 'Union Square');
+    var endLineUnionSqIndex = this.getStationIndex(endLine, 'Union Square');
+
```

Comment:
This `planTrip()` function has a bunch of functions defined within it... is this intentional or did you leave out a `}` somewhere?